2335.minimum-amount-of-time-to-fill-cups.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/minimum-amount-of-time-to-fill-cups/
+ */
+
+/**
+ * @param {number[]} amount
+ * @return {number}
+ */
+var fillCups = function(amount) {
+    
+    //Sort numbers from highest to lowest
+    let sortedNumber = amount.sort((a, b) => b - a);
```

Comment:
```suggestion
    const sortedNumbers = amount.sort((a, b) => b - a);
```
Const is better
"sortedNumbers" makes more sense

Reply:
This might cause side effects.
It is better to make a copy of "amount" if you want to manipulate elements of the array.